services/core-api/app/api/projects/major_mine_application/models/major_mine_application.py

```diff
@@ -64,6 +65,32 @@ def find_by_project_guid(cls, _id):
         except ValueError:
             return None
 
+    def send_mma_submit_email(self):
+        recipients = [contact.email for contact in self.project.contacts]
+        primary_documents = [document.document_name for document in self.documents if document.major_mine_application_document_type_code == "PRM"]
+        spatial_documents = [document.document_name for document in self.documents if document.major_mine_application_document_type_code == "SPT"]
+        supporting_documents = [document.document_name for document in self.documents if document.major_mine_application_document_type_code == "SPR"]
+
+        def generate_list_element(element):
+            return f'<li>{element}</li>'
+
+        # TODO: Update this link(add additional path traversal) when Steps(1-3) are hooked up for MMA on the frontend
+        link = f'{Config.MINESPACE_PRODUCTION_URL}/projects/{self.project_guid}/major-mine-application/{self.major_mine_application_guid}'
+
+        subject = f'Major Mine Application Submitted for {self.project.project_title}'
+        body = '<p>The following documents have been submitted with this Major Mine Application:</p>'
+        body += '<p>Primary document(s):</p>'
+        body += f'<ul>{"".join(list(map(generate_list_element, primary_documents)))}</ul>'
```

Comment:
do you not need the same check here as you do with the other docs or is there a guaranteed primary doc?

Reply:
You need at least one primary document to create a Major Mine Application.